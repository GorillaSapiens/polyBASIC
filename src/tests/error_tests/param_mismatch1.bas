DEF a(x) 12
b = a(5,6)
