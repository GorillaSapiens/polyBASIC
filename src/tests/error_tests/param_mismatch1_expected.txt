ERROR: @2:6, FUNCTION DEFINITION PARAMETER MISMATCH ❮a❯ 1<>2
